analysis/wwz/make_datacards.py

```diff
@@ -27,6 +27,10 @@
     "btagSFlight_uncorrelated_2018"    : {"yr_rel":"UL18", "yr_notrel": ["UL16APV", "UL16", "UL17"]},
     "btagSFbc_uncorrelated_2018"       : {"yr_rel":"UL18", "yr_notrel": ["UL16APV", "UL16", "UL17"]},
 }
+SYSTS_SPECIAL_RUN3 = {
```

Comment:
I think we should either rename SYSTS_SPECIAL as SYSTS_SPECIAL_RUN2 for more symmetry in the way things are handled